nltk/test/unit/test_concordance.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+
+import unittest
+from nose import with_setup
+
+from nltk.corpus import gutenberg
+from nltk.text import Text
+
+import contextlib
+import sys
+
+try:
+    from StringIO import StringIO
+except ImportError as e:
+    from io import StringIO
+
+
+@contextlib.contextmanager
+def stdout_redirect(where):
+    sys.stdout = where
+    try:
+        yield where
+    finally:
+        sys.stdout = sys.__stdout__
+
+
+def clean(raw_str):
+    """Remove spaces and case"""
+    return raw_str.lower().replace(" ", "")
+
+
+def clean_list(input):
+    """Removes spaces and case in list elements"""
+    return [clean(raw_str) for raw_str in input]
+
```

Comment:
invisible white space, inconsistent new lines, and the like. Will try to refactor into something that doesn't need cleaning.